go/teams/loader_ctx.go

```diff
@@ -199,27 +210,39 @@ func perUserEncryptionKey(m libkb.MetaContext, userSeqno keybase1.Seqno) (*libkb
 	return kr.GetEncryptionKeyBySeqnoOrSync(m, userSeqno)
 }
 
+func (l *LoaderContextG) merkleLookupWithHidden(ctx context.Context, teamID keybase1.TeamID, public bool, harg *libkb.LookupTeamHiddenArg) (r1 keybase1.Seqno, r2 keybase1.LinkID, hiddenIsFresh bool, err error) {
+	leaf, err := l.G().GetMerkleClient().LookupTeamWithHidden(l.MetaContext(ctx), teamID, harg)
+	r1, r2, hiddenIsFresh, err = l.processMerkleReply(ctx, teamID, public, leaf, err)
+	return r1, r2, hiddenIsFresh, err
+}
+
 func (l *LoaderContextG) merkleLookup(ctx context.Context, teamID keybase1.TeamID, public bool) (r1 keybase1.Seqno, r2 keybase1.LinkID, err error) {
 	leaf, err := l.G().GetMerkleClient().LookupTeam(l.MetaContext(ctx), teamID)
-	if err != nil {
-		return r1, r2, err
+	r1, r2, _, err = l.processMerkleReply(ctx, teamID, public, leaf, err)
+	return r1, r2, err
+}
+
+func (l *LoaderContextG) processMerkleReply(ctx context.Context, teamID keybase1.TeamID, public bool, leaf *libkb.MerkleTeamLeaf, ein error) (r1 keybase1.Seqno, r2 keybase1.LinkID, hiddenIsFresh bool, err error) {
```

Comment:
don't understand this ein pattern (is it `error in`?). is it to shorten the code?

Reply:
ok, removed.